### DISCUSSION TITLE
unable to fetch credentials from jenkins

### DISCUSSION THREAD

**Discussion by surendraalamuru22** [learndevopsonline/learndevopsonline#1861] [Q&A]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/159264263/c828d112-e6b7-487d-889d-489f48530a8c)

jenkins sever username:admin, password:admin123

**Comment by chaithanya1812** [comment]
please join the lab session once..

**Reply by surendraalamuru22** [reply]
ok chaitanya

**Reply by chaithanya1812** [reply]
Hi @surendraalamuru22 , will you join?

**Reply by surendraalamuru22** [reply]
ya i will join

**Comment by chaithanya1812** [comment]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/20cf85a5-6fd3-4265-b1f6-221e3d709002)

**Reply by surendraalamuru22** [reply]
secret is there in aws from that i need to fetch and store them inside jenkins credentials

**Comment by chaithanya1812** [comment]
Hi @surendraalamuru22 ,may I know your batch details, like
which batch you belongs to?

**Reply by surendraalamuru22** [reply]
b67 chaitanya

**Reply by chaithanya1812** [reply]
oky, will you join lab session once , so that we can clear the issue
here is the link
https://learndevopsonline.my.canva.site/lab-help

**Comment by surendraalamuru22** [comment]
its clear now
